doc/implementation

```diff
@@ -3,9 +3,11 @@ Implementation:
 C/C++ handling:
 ==============
 
+NOTE: This document describes legacy abrt-hook-cpp that was removed in favour of systemd-coredump.
+
 We change /proc/sys/kernel/core_pattern to invoke abrtd helper to save
 the coredump of the crashing app:
-* helper source code: http://git.fedorahosted.org/git/abrt.git?p=abrt.git;a=blob_plain;f=src/Hooks/abrt-hook-python.cpp
+* helper source code:
```

Comment:
So, given that this is empty now, maybe we should get rid of it altogether? The other option is linking to a commit prior to removal, but we could just do this in one go.

Reply:
Right, the rest of the document relies heavily on the existence of that thing.

Reply:
Feel free to ignore my comments, because everything would need to be removed or rewritten.